appyters/ChEA3_Appyter/ChEA3_Appyter.ipynb

```diff
@@ -0,0 +1,770 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "10fc200c",
+   "metadata": {},
+   "source": [
+    "# ChEA3 Appyter\n",
+    "\n",
+    "\n",
+    "The ChEA3 Appyter predicts transcription factors (TFs) associated with user-input sets of genes. Discrete query gene sets are compared to ChEA3 libraries of TF target gene sets assembled from multiple orthogonal 'omics' datasets. The Fisher's Exact Test, with a background size of 20,000, is used to compare the input gene set to the TF target gene sets in order to determine which TFs may be most closely associated with the input gene set. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6bfa4157",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%% appyter init\n",
```

Comment:
Please make this the first cell in the appyter and make it exactly:
```
#%%appyter init
from appyter import magic
magic.init(lambda _=globals: _())
```

I.e. no space after %%